M2/Macaulay2/packages/ThinSincereQuivers.m2

```diff
@@ -1,20 +1,18 @@
 newPackage(
     "ThinSincereQuivers",
     Headline => "Construction of flow polytopes and their associated quivers",
-    Version => "0.0",
-    Date => "November 20, 2019",
+    Version => "0.1",
+    Date => "January 18, 2025",
     Authors => {
         {Name => "Mary Barker",
-         Email => "marybarker@wustl.edu",
+         Email => "marybarker103@gmail.com",
          HomePage => "https://github.com/marybarker"}, 
         {Name => "Patricio Gallardo",
          Email => "pgallard@ucr.edu",
          HomePage => "http://patriciogallardo.com/"
         }
     },
-    PackageImports => {"Graphs", "Polyhedra", "LatticePolytopes"},
-    PackageExports => {"Graphs", "Polyhedra"},
-    Keywords => {"Toric Geometry"}
```

Comment:
The tests are failing because this PR is removing the `Keywords` option from the call to `newPackage`.  It was previously `Keywords => {"Toric Geometry"}`.

Reply:
Oh, thanks! That makes sense.